examples/deploy/terraform/cluster/main.tf

```diff
@@ -7,14 +7,15 @@ data "terraform_remote_state" "infra" {
 }
 
 locals {
-  infra = data.terraform_remote_state.infra.outputs.infra
-  kms   = var.kms_info != null ? var.kms_info : local.infra.kms
+  infra  = data.terraform_remote_state.infra.outputs.infra
+  kms    = var.kms_info != null ? var.kms_info : local.infra.kms
+  region = var.region != null ? var.region : local.infra.region
```

Comment:
I am not sure we want this here, given that we need to deploy in the region where infra was created